tests/CopyCommandTest.php

```diff
@@ -0,0 +1,77 @@
+{
+    /** @test */
+    public function component_is_copied_by_copy_command()
+    {
+        Artisan::call('make:livewire bob');
+
+        $this->assertTrue(File::exists($this->livewireClassesPath('Bob.php')));
+        $this->assertTrue(File::exists($this->livewireViewsPath('bob.blade.php')));
+
+        Artisan::call('livewire:cp bob lob');
+
+        $this->assertTrue(File::exists($this->livewireClassesPath('Lob.php')));
+        $this->assertTrue(File::exists($this->livewireViewsPath('lob.blade.php')));
+
+        $this->assertTrue(File::exists($this->livewireClassesPath('Bob.php')));
+        $this->assertTrue(File::exists($this->livewireViewsPath('bob.blade.php')));
+    }
+
+    /** @test */
+    public function nested_component_is_copied_by_copy_command()
+    {
+        Artisan::call('make:livewire bob.lob');
+
+        $this->assertTrue(File::exists($this->livewireClassesPath('Bob/Lob.php')));
+        $this->assertTrue(File::exists($this->livewireViewsPath('bob/lob.blade.php')));
+
+        Artisan::call('livewire:cp bob.lob bob.lob.law');
+
+        $this->assertTrue(File::exists($this->livewireClassesPath('Bob/Lob/Law.php')));
+        $this->assertTrue(File::exists($this->livewireViewsPath('bob/lob/law.blade.php')));
+
+        $this->assertTrue(File::exists($this->livewireClassesPath('Bob/Lob.php')));
+        $this->assertTrue(File::exists($this->livewireViewsPath('bob/lob.blade.php')));
+    }
+
+    /** @test */
+    public function multiword_component_is_copied_by_copy_command()
+    {
+        Artisan::call('make:livewire bob-lob');
+
+        $this->assertTrue(File::exists($this->livewireClassesPath('BobLob.php')));
+        $this->assertTrue(File::exists($this->livewireViewsPath('bob-lob.blade.php')));
+
+        Artisan::call('livewire:cp bob-lob lob-law');
+
+        $this->assertTrue(File::exists($this->livewireClassesPath('/LobLaw.php')));
+        $this->assertTrue(File::exists($this->livewireViewsPath('lob-law.blade.php')));
+
+        $this->assertTrue(File::exists($this->livewireClassesPath('BobLob.php')));
+        $this->assertTrue(File::exists($this->livewireViewsPath('bob-lob.blade.php')));
+    }
+
+    /** @test */
+    public function pascal_case_component_is_automatically_converted_by_copy_command()
+    {
+        Artisan::call('make:livewire BobLob.BobLob');
+
+        $this->assertTrue(File::exists($this->livewireClassesPath('BobLob/BobLob.php')));
+        $this->assertTrue(File::exists($this->livewireViewsPath('bob-lob/bob-lob.blade.php')));
+
+        Artisan::call('livewire:cp BobLob.BobLob LobLaw.LobLaw');
+
+        $this->assertTrue(File::exists($this->livewireClassesPath('BobLob/BobLob.php')));
+        $this->assertTrue(File::exists($this->livewireViewsPath('bob-lob/bob-lob.blade.php')));
+
+        $this->assertTrue(File::exists($this->livewireClassesPath('LobLaw/LobLaw.php')));
+        $this->assertTrue(File::exists($this->livewireViewsPath('lob-law/lob-law.blade.php')));
+    }
```

Comment:
Add test for "cant copy component to a name that already exists"

Reply:
Add force option for this command

Reply:
Make a phpunit run group for just showing command output

Reply:
I have added the --force option to the copy command and added tests
- "cannot copy component to a name that already exists"
- "can copy component to a name that already exists if forced" tests

I'm not sure how to do the command output phpunit thing